doc/source/data_models/data-atm.rst

```diff
@@ -24,42 +24,47 @@ These variables will appear in ``env_run.xml`` and the resulting values are comp
-   "DATM_TOPO",             "Surface topography"
-   "DATM_CO2_TSERIES",      "CO2 time series type"
-   "DATM_CPLHIST_CASE",     "Coupler history data mode case name" 
-   "DATM_CPLHIST_DIR",      "Coupler history data mode directory containing coupler history data" 
-   "DATM_CPLHIST_YR_ALIGN", "Coupler history data model simulation year corresponding to data starting year" 
-   "DATM_CPLHIST_YR_START", "Coupler history data model starting year to loop data over" 
-   "DATM_CPLHIST_YR_END",   "Coupler history data model ending year to loop data over" 
-   "DATM_CLMNCEP_YR_ALIGN", "I compsets only - simulation year corresponding to data starting year" 
-   "DATM_CPLHIST_YR_START", "I compsets only - data model starting year to loop data over" 
-   "DATM_CPLHIST_YR_END",   "I compsets only - data model ending year to loop data over" 
+   "DATM_MODE",                "Mode for atmospheric component"
+   "",                         "Valid values are: CORE2_NYF,CORE2_IAF,CLM_QIAN,CLM_QIAN_WISO,CLM1PT,CLMCRUNCEP,"
+   "",                         "CLMCRUNCEP_V5,CLMGSWP3,WW3,CPLHIST"
+
+   "DATM_PRESAERO",            "Optional prescribed aerosol forcing"
+   "DATM_TOPO",                "Optional Surface topography"
+   "DATM_CO2_TSERIES",         "Optional CO2 time series type"
+
+   "DATM_CPLHIST_DOMAIN_FILE", "Coupler history forcing data mode full pathname model domain file "
+   "DATM_CPLHIST_CASE",        "Coupler history forcing data mode case name"
+   "DATM_CPLHIST_DIR",         "Coupler history forcing data mode directory containing coupler history data"
+   "DATM_CPLHIST_YR_ALIGN",    "Coupler history forcing data model simulation year corresponding to data starting year"
+   "DATM_CPLHIST_YR_START",    "Coupler history forcing data model starting year to loop data over"
+   "DATM_CPLHIST_YR_END",      "Coupler history forcing data model ending year to loop data over"
+
+   "DATM_CLMNCEP_YR_ALIGN",    "I compsets only - simulation year corresponding to data starting year"
+   "DATM_CLMNCEP_YR_START",    "I compsets only - data model starting year to loop data over"
+   "DATM_CLMNCEP_YR_END",      "I compsets only - data model ending year to loop data over"
+
+.. note:: If ``DATM_MODE`` is set to ``CPLHIST``, it is normally assumed that the model domain will be identical to **all** of the stream domains. To ensure this, the xml variables ``ATM_DOMAIN_PATH`` and ``ATM_DOMAIN_FILE`` are ignored and a valid setting **must be given** for ``DATM_CPLHIST_DOMAIN_FILE``. If ``DATM_CPLHIST_DOMAIN_FILE`` is set to ``null``, then the datm component domain information is read in from the first coupler history file in the target stream and  it is assumed that the first coupler stream file that is pointed to contains the domain  information for that stream. This is the default mode that should be used for this mode. Alternatively, ``DATM_CPLHIST_DOMAIN_FILE`` can be set to ``$ATM_DOMAIN_PATH/$ATM_DOMAIN_FILE`` in a non-default configuration.
```

Comment:
In another discussion, it was thought that `'none'` is more user accessible than `'null'`. Is it possible to use `'none'` in this context?

Reply:
Unfortunately, this is not really possible in this case. A 'null' value is hard-wired as the default value in the data model shr strdata input namelist - and it would be good to be consistent with the code. This could be changed -but I'm really wary to do this now.